helper/Makefile.am

```diff
@@ -60,6 +60,6 @@ __LIB__libodphelper_la_SOURCES += \
 				linux/thread.c
 endif
 
-__LIB__libodphelper_la_LIBADD = $(PTHREAD_LIBS) $(LIB)/libodp-linux.la
+__LIB__libodphelper_la_LIBADD = $(PTHREAD_LIBS)
```

Comment:
I'm not so sure here. Odphelper uses symbols from odp, so it should be linked with it.

Reply:
Then in the application linking we do link both libodp-helper and libodp-linux. That results in libodp-linux.la being linked twice.

Reply:
This sounds suspicious. Both static and dynamic linking should handle this correctly.

Reply:
The way the current automake file is translated to a linker command is that the linker command contains libodp-linux.a twice when we link the applications.
Are you suggesting that automake (or libtool) should detect multiple definitions of the same library and include it only once? That I was not aware in automake. Rather in my experience it does not perform any such detects.

Also it seems strange to me to add libodp-linux here because it would indicate that you could be allowed to link against libodp-helper using one libodp-linux and then another version of libodp-linux. Rather I believe libodp-helper should complain if someone tries to link against it and does not provide libodp-linux explicitly.